benchmark-timed-schedule.yaml

```diff
@@ -0,0 +1,323 @@
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: google/cloud-sdk
+        params:
+          <<: *common_infrastructure_params
+          IMPORT_EXISTING_PROJECT: ((import_existing_project))
+        inputs:
+          - name: eq-terraform-load-generator
+        run:
+          path: bash
+          args:
+            - -exc
+            - |
+              apt-get install -y unzip
+
+              git clone https://github.com/kamatama41/tfenv.git ~/.tfenv && \
+              ln -s /root/.tfenv/bin/* /usr/local/bin
+
+              cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+              $SERVICE_ACCOUNT_JSON
+              EOL
+
+              gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+
+              cd eq-terraform-load-generator
+
+              tfenv install
+
+              ./scripts/deploy_infrastructure.sh
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Infrastructure Deploy Failed
+            color: danger
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+    on_success:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Infrastructure Deploy Passed
+            color: good
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+
+##############################################################################
+# Run Benchmark
+##############################################################################
+
+  - name: run-benchmark
+    serial_groups: [run-benchmark]
+    plan:
+    - get: eq-survey-runner-benchmark
+    - get: scheduled-benchmark-trigger
+      passed: [build-survey-runner-benchmark, deploy-infrastructure]
+      trigger: true
+    - task: Run Survey Runner Benchmark
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: theorf/google-cloud-sdk-helm
+        params:
+          SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        inputs:
+          - name: eq-survey-runner-benchmark
+        run:
+          path: bash
+          args:
+            - -exc
+            - |
+              apt-get install -y procps
+
+              cat >~/gcloud-service-key.json <<EOL
+              $SERVICE_ACCOUNT_JSON
+              EOL
+
+              gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+
+              cd eq-survey-runner-benchmark
+
+              helm init --client-only
+              helm plugin install https://github.com/rimusz/helm-tiller
+
+              PROJECT_ID=($(gcloud projects list --filter=name="((project_name))" --format="value(PROJECT_ID)"))
```

Comment:
```
              PROJECT_ID=($(gcloud projects list --filter=project_id="((project_id))" --format="value(PROJECT_ID)"))
```

Reply:
Have now changed this to filter based on $IMPORT_EXISTING_PROJECT.